PsfLauncher/main.cpp

```diff
@@ -25,12 +63,53 @@ int launcher_main(PWSTR args, int cmdShow) noexcept try
     auto exeName = appConfig->get("executable").as_string().wide();
     auto dirPtr = appConfig->try_get("workingDirectory");
     auto dirStr = dirPtr ? dirPtr->as_string().wide() : nullptr;
+	auto monitor = PSFQueryAppMonitorConfig();
 
     // At least for now, configured launch paths are relative to the package root
     std::filesystem::path packageRoot = PSFQueryPackageRootPath();
     auto exePath = packageRoot / exeName;
     std::wstring cmdLine = L"\"" + exePath.filename().native() + L"\" " + args;
 
+if (monitor != nullptr )

```

Comment:
Formatting

Reply:
Addressed in commit 727a9e9856d361f686523f2ea2907876c3ef9b23 on 11/28